libs/libarchfpga/src/physical_types.h

```diff
@@ -619,6 +619,9 @@ struct t_physical_tile_type {
 
     /* Returns the indices of pins that contain a clock for this physical logic block */
     std::vector<int> get_clock_pins_indices() const;
+
+    bool is_input_type;
```

Comment:
Should at least comment these (what do they mean)?
Also say most code should try not to use and be generic for all block types if possible (could deprecate at that point).

Reply:
Keith can you do a grep to figure out how many call sites that use this now.
Could we get rid of these variables?

Reply:
I have no strong opinion about changing these, but would rather get rid of them.

Reply:
Comments in and issue filed